### PR TITLE
minor doc fixes

### PR DESCRIPTION
**Pull request by knarz**
I started a first pass of the documentation in order to get a rough overview of overall design of the system and fixed some minor things. Hope they make sense.

### CODE REVIEW

**Comment by mhluongo** [comment]
Thanks for the PR! Giving it a look :)

**Comment by knarz** [comment]
Cool, if it is useful, then I'll just submit more whenever I read the rest. I assume the discord would be the best place to ask for clarifications?

File: docs/bonding/index.adoc [hunk 1]
```diff
@@ -123,7 +123,7 @@ deployment of the mechanism.
 [example]
 Let 1 BTC be worth $10,000, and 1 ETH be worth $200. Signers have to put up 75
 ETH to back a deposit. Signers are expected to make a signer fee of 5 basis
-points on a $10,000 deposit for $15,000 of collateral (150% of $10,000): $5.
+points on a $10,000 deposit for $15,000 of collateral (0.05% of $10,000): $5.
```
[the remark this answers is not in the document]
Comment (knarz) [reply docs/bonding/index.adoc:126]: Oh, maybe `$15,000 (150% of $10,000) of collateral` instead then?

File: docs/signer-fees/index.adoc [hunk 2]
```diff
@@ -54,7 +54,7 @@ It is assumed that each `Signer` contributes equally to the collateralization of
 a `Deposit`.
 
 The capital cost per `Signer` is `BondValue / N`. Using `LotSize = 1
-BTC` and `OverCollateralizationFactor = 150%`, that is `1.5 BTC / N`.
+BTC` and `OverCollateralizationFactor = 1.5`, that is `1.5 BTC / N`.
```
[the remark this answers is not in the document]
Comment (knarz) [reply docs/signer-fees/index.adoc:57]: To me, a factor is a divider/multiplier without a unit but that might just be personal preference again

File: docs/bonding/index.adoc [hunk 3]
```diff
@@ -174,8 +174,8 @@ long periods of time.
 [[liquidation]]
 === Liquidation
 
-:fraudulent-signature: footnote:[Fraudulent signatures are signatures not \
-explicitly authorized by the tBTC system. The system only authorizes redemption \
+:fraudulent-signature: footnote:[signatures not explicitly authorized by the \
+tBTC system are considered fraudulent. The system only authorizes redemption \
```
[the remark this answers is not in the document]
Comment (knarz) [reply docs/bonding/index.adoc:178]: Was just trying to get rid of the »signatures are signatures« formulation

**Comment by knarz** [comment]
I'll just close this one then and look at #645 instead

**Pull request closed without merging**